features/ASM000_Assemblies-information.feature

```diff
@@ -0,0 +1,21 @@
+@implementer-agreement
+@ASM
+@version1
+@E00020
+
+Feature: ASM000 - Assemblies information
+    The rule verifies the presence of elements composed of / constructed by other elements.
```

Comment:
```suggestion
    The rule verifies the presence of IFC entities used to model elements composed of / constructed by other elements.
```
It's the same structure, over and over and over again.
Take this sentence "The rule verifies the presence of IFC entities used to" and append it before the verb of each description - in this case "...model elements composed of..."